src/fun/counting/main.go.tmpl

```diff
@@ -0,0 +1,89 @@
+{{/*
+        Counting command. Count and have fun!
+        
+        Author: H1nr1 <https://github.com/H1nr1>
+*/}}
+
+{{/* Configurable Values */}}
+{{ $Admin := 123456789 }} {{/* Administrator role ID; EDIT THIS TO A ROLE ID YOU HAVE */}}
+{{/* Optional */}}
+{{ $CountTwice := true }} {{/* Allow users to count multiple times in a row; true/false */}}
+{{ $RoleID := false }} {{/* Counting role ID; set to false to disable */}}
+{{ $SecondChance := true }} {{/* Second chance if wrong; true/false */}}
+{{ $Leaderboard := true }} {{/* If you added the leaderboard CC; true/false */}}
+{{ $CorrectEmoji := "✅" }} {{/* Emoji to react with if number is correct; if custom, use format name:id */}}
+{{ $WarningEmoji := "⚠️" }} {{/* Emoji to react with if wrong number AND Second Chance is true/on; if custom, use format name:id */}}
+{{ $IncorrectEmoji := "❌" }} {{/* Emoji to react with if number is incorrect; if custom, use format name:id */}}
+{{/* End of configuration values */}}
+
+{{/* No Touchy */}}
+{{ with .ExecData }}
+	{{ if not (getMessage (toInt (dbGet 0 "Channel").Value) .) }} {{/* Check if number was deleted */}}
+		{{ sendMessage (toInt (dbGet 0 "Channel").Value) (cembed 
+			"description" (print "<@" (dbGet 0 "LastUser").Value "> deleted their number which was correct!\nThe next number is " (dbGet 0 "Next").Value)
+			"color" 30654) }}
+	{{ return }}{{ end }}
+{{ end }}
+
+{{/* Initializing variables */}}
+{{ $LastUser := (dbGet 0 "LastUser").Value }}{{ $Next := (dbGet 0 "Next").Value }}
+{{ $Number := "" }}{{ $Number = round (slice ($Number = (exec "calc" (index .Args 0))) 9 (len $Number | add -1)) }}
+{{ $Wrong := "" }}{{ if $SecondChance }}{{ $Wrong = (dbGet 0 "CSecondChance").Value }}{{ end }}{{ $Silence := "" }}
+
+{{ if and (hasRoleID $Admin) (eq (toInt $Number) 0) (lt (toInt (dbGet 0 "Admin").Value) 1) }}
+	{{ dbSet 0 "Channel" (toString .Channel.ID) }}{{ dbSet 0 "LastUser" (toString .User.ID) }}{{ dbSet 0 "Next1" 1 }}{{ dbIncr 0 "Admin" 1 }}{{ addReactions $CorrectEmoji }}
+{{ return }}{{ end }}
+
+{{ if or (ne (toString $LastUser) (toString .User.ID)) $CountTwice }} {{/* Checks user */}}
+	{{ if eq (toInt $Next) (toInt $Number) }} {{/* Checks if correct number */}}
+		{{ addReactions $CorrectEmoji }}
+		{{ dbSet 0 "Next" (add $Next 1) }}
+		{{ dbSet 0 "LastUser" (toString .User.ID) }}
+		{{ if not $RoleID }} {{/* Counting role */}}
+			{{ takeRoleID $LastUser $RoleID }}
```

Comment:
```suggestion
			{{ takeRoleID $LastUser $RoleID 1 }}
```
Add a specific delay, else it will default to 10 seconds and cause confusion amongst the users with that role.